### DISCUSSION TITLE
Is it possible to point to existing database? How?

### DISCUSSION THREAD

**Discussion by AshfaqMemon** [HCL-TECH-SOFTWARE/connections-automation#132] [Q&A]
Hi

We already have installed and configured MSSQL Server 2016. I want to know if we can use existing database rather than installing fresh database.

**Comment by sabrina-yee** [comment]
Release [v0.5.2](https://github.com/HCL-TECH-SOFTWARE/connections-automation/releases/tag/v0.5.2) includes example inventories and a playbook to deploy on existing database.
